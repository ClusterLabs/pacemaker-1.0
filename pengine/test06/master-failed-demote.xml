 <cib generated="true" admin_epoch="0" epoch="1" num_updates="28" have_quorum="true" ignore_dtd="false" num_peers="2" ccm_transition="2" cib_feature_revision="1.3" dc_uuid="888e539a-c6ef-496d-b79a-77cbf0f9e5e4" cib-last-written="Thu Apr 17 13:20:12 2008">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options">
         <attributes>
           <nvpair id="cib-bootstrap-options-dc-version" name="dc-version" value="2.1.3-node: 552305612591183b1628baa5bc6e903e0f1e26a3"/>
         </attributes>
       </cluster_property_set>
       <cluster_property_set id="set01">
         <attributes>
           <nvpair id="default-resource-stickiness" name="default-resource-stickiness" value="INFINITY"/>
           <nvpair id="cib-bootstrap-options-default-resource-failure-stickiness" name="default-resource-failure-stickiness" value="-INFINITY"/>
           <nvpair id="stonith-enabled" name="stonith-enabled" value="false"/>
           <nvpair id="stonith-action" name="stonith-action" value="reboot"/>
           <nvpair id="cluster-delay" name="cluster-delay" value="180s"/>
           <nvpair id="default-action-timeout" name="default-action-timeout" value="120s"/>
           <nvpair id="no-quorum-policy" name="no-quorum-policy" value="ignore"/>
           <nvpair id="symmetric-cluster" name="symmetric-cluster" value="true"/>
           <nvpair id="short-resource-names" name="short-resource-names" value="true"/>
           <nvpair id="is-managed-default" name="is-managed-default" value="true"/>
           <nvpair id="stop-orphan-resources" name="stop-orphan-resources" value="true"/>
           <nvpair id="stop-orphan-actions" name="stop-orphan-actions" value="true"/>
           <nvpair id="remove-after-stop" name="remove-after-stop" value="false"/>
           <nvpair id="dc-deadtime" name="dc-deadtime" value="10s"/>
           <nvpair id="cluster-recheck-interval" name="cluster-recheck-interval" value="0"/>
           <nvpair id="election-timeout" name="election-timeout" value="2min"/>
           <nvpair id="shutdown-escalation" name="shutdown-escalation" value="20min"/>
           <nvpair id="crmd-integration-timeout" name="crmd-integration-timeout" value="3min"/>
           <nvpair id="crmd-finalization-timeout" name="crmd-finalization-timeout" value="10min"/>
           <nvpair id="pe-error-series-max" name="pe-error-series-max" value="-1"/>
           <nvpair id="pe-warn-series-max" name="pe-warn-series-max" value="-1"/>
           <nvpair id="pe-input-series-max" name="pe-input-series-max" value="-1"/>
           <nvpair id="startup-fencing" name="startup-fencing" value="true"/>
         </attributes>
       </cluster_property_set>
     </crm_config>
     <nodes>
       <node id="888e539a-c6ef-496d-b79a-77cbf0f9e5e4" uname="dl380g5b" type="normal"/>
       <node id="3831f23f-da0a-470e-b8e7-16e61a18c3c3" uname="dl380g5a" type="normal"/>
     </nodes>
     <resources>
       <master_slave id="ms-sf" notify="true">
         <instance_attributes id="ms-sf-attr">
           <attributes>
             <nvpair id="ms-sf-clone_max" name="clone_max" value="2"/>
             <nvpair id="ms-sf-clone_node_max" name="clone_node_max" value="1"/>
             <nvpair id="ms-sf-master_max" name="master_max" value="1"/>
             <nvpair id="ms-sf-master_node_max" name="master_node_max" value="1"/>
           </attributes>
         </instance_attributes>
         <group id="group">
           <primitive id="stateful-1" class="ocf" type="Stateful" provider="heartbeat">
             <operations>
               <op id="start-1" name="start" timeout="60s" on_fail="stop"/>
               <op id="monitor-1m" name="monitor" timeout="60s" on_fail="stop" interval="10" role="Master"/>
               <op id="monitor-1s" name="monitor" timeout="60s" on_fail="stop" interval="20" role="Slave"/>
               <op id="stop-1" name="stop" timeout="60s" on_fail="stop"/>
             </operations>
           </primitive>
           <primitive id="stateful-2" class="ocf" type="Stateful" provider="heartbeat">
             <operations>
               <op id="start-2" name="start" timeout="60s" on_fail="stop"/>
               <op id="monitor-2m" name="monitor" timeout="60s" on_fail="stop" interval="10" role="Master"/>
               <op id="monitor-2s" name="monitor" timeout="60s" on_fail="stop" interval="20" role="Slave"/>
               <op id="stop-2" name="stop" timeout="60s" on_fail="stop"/>
             </operations>
           </primitive>
         </group>
       </master_slave>
     </resources>
     <constraints>
       <rsc_location id="rsc_location" rsc="ms-sf">
         <rule id="location-0" role="Master" score="100">
           <expression id="exp-0" attribute="#uname" operation="eq" value="dl380g5a"/>
         </rule>
         <rule id="location-1" role="Master" score="500">
           <expression id="exp-1" attribute="#uname" operation="eq" value="dl380g5b"/>
         </rule>
       </rsc_location>
     </constraints>
   </configuration>
   <status>
     <node_state id="888e539a-c6ef-496d-b79a-77cbf0f9e5e4" uname="dl380g5b" crmd="online" crm-debug-origin="do_update_resource" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="888e539a-c6ef-496d-b79a-77cbf0f9e5e4">
         <lrm_resources>
           <lrm_resource id="stateful-1:0" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-1:0_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="3:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:7;3:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="2" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1:0_start_0" operation="start" crm-debug-origin="do_update_resource" transition_key="13:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;13:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="6" crm_feature_set="2.0" rc_code="0" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1:0_promote_0" operation="promote" crm-debug-origin="do_update_resource" transition_key="14:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;14:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="9" crm_feature_set="2.0" rc_code="0" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1:0_demote_0" operation="demote" crm-debug-origin="do_update_resource" transition_key="10:5:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:8;10:5:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="12" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
           <lrm_resource id="stateful-1:1" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-1:1_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="5:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:7;5:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="4" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
           <lrm_resource id="stateful-2:0" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-2:0_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="4:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:7;4:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="3" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2:0_start_0" operation="start" crm-debug-origin="do_update_resource" transition_key="16:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;16:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="7" crm_feature_set="2.0" rc_code="0" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2:0_promote_0" operation="promote" crm-debug-origin="do_update_resource" transition_key="17:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;17:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="8" crm_feature_set="2.0" rc_code="0" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2:0_stop_0" operation="stop" crm-debug-origin="do_update_resource" transition_key="3:5:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;3:5:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="13" crm_feature_set="2.0" rc_code="0" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
           <lrm_resource id="stateful-2:1" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-2:1_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="6:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:7;6:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="5" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="888e539a-c6ef-496d-b79a-77cbf0f9e5e4">
         <instance_attributes id="status-888e539a-c6ef-496d-b79a-77cbf0f9e5e4">
           <attributes>
             <nvpair id="status-888e539a-c6ef-496d-b79a-77cbf0f9e5e4-probe_complete" name="probe_complete" value="true"/>
             <nvpair id="status-888e539a-c6ef-496d-b79a-77cbf0f9e5e4-fail-count-stateful-2:0" name="fail-count-stateful-2:0" value="1"/>
             <nvpair id="status-888e539a-c6ef-496d-b79a-77cbf0f9e5e4-fail-count-stateful-1:0" name="fail-count-stateful-1:0" value="1"/>
           </attributes>
         </instance_attributes>
       </transient_attributes>
     </node_state>
     <node_state id="3831f23f-da0a-470e-b8e7-16e61a18c3c3" uname="dl380g5a" crmd="online" crm-debug-origin="do_update_resource" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="3831f23f-da0a-470e-b8e7-16e61a18c3c3">
         <lrm_resources>
           <lrm_resource id="stateful-1:1" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-1:1_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="10:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:7;10:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="4" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1:1_start_0" operation="start" crm-debug-origin="do_update_resource" transition_key="23:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;23:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="6" crm_feature_set="2.0" rc_code="0" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1:1_monitor_20000" operation="monitor" crm-debug-origin="do_update_resource" transition_key="24:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;24:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="8" crm_feature_set="2.0" rc_code="0" op_status="0" interval="20000" op_digest="873ed4f07792aa8ff18f3254244675ea"/>
           </lrm_resource>
           <lrm_resource id="stateful-2:0" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-2:0_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="9:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:7;9:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="3" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
           <lrm_resource id="stateful-2:1" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-2:1_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="11:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:7;11:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="5" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2:1_start_0" operation="start" crm-debug-origin="do_update_resource" transition_key="25:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;25:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="7" crm_feature_set="2.0" rc_code="0" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2:1_monitor_20000" operation="monitor" crm-debug-origin="do_update_resource" transition_key="26:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:0;26:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="9" crm_feature_set="2.0" rc_code="0" op_status="0" interval="20000" op_digest="873ed4f07792aa8ff18f3254244675ea"/>
           </lrm_resource>
           <lrm_resource id="stateful-1:0" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-1:0_monitor_0" operation="monitor" crm-debug-origin="do_update_resource" transition_key="8:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition_magic="0:7;8:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call_id="2" crm_feature_set="2.0" rc_code="7" op_status="0" interval="0" op_digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="3831f23f-da0a-470e-b8e7-16e61a18c3c3">
         <instance_attributes id="status-3831f23f-da0a-470e-b8e7-16e61a18c3c3">
           <attributes>
             <nvpair id="status-3831f23f-da0a-470e-b8e7-16e61a18c3c3-probe_complete" name="probe_complete" value="true"/>
           </attributes>
         </instance_attributes>
       </transient_attributes>
     </node_state>
   </status>
 </cib>
