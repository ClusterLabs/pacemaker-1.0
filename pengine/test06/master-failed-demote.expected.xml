<?xml version="1.0" encoding="UTF-8"?>
<cib admin_epoch="0" epoch="1" num_updates="28" dc-uuid="888e539a-c6ef-496d-b79a-77cbf0f9e5e4" have-quorum="true" remote-tls-port="0" validate-with="pacemaker-1.0">
   <configuration>
     <crm_config>
       <cluster_property_set id="cib-bootstrap-options"><nvpair id="nvpair" name="dc-version" value="2.1.3-node: 552305612591183b1628baa5bc6e903e0f1e26a3"/></cluster_property_set>
       <cluster_property_set id="set01"><nvpair id="nvpair" name="default-resource-stickiness" value="INFINITY"/><nvpair id="nvpair" name="default-resource-failure-stickiness" value="-INFINITY"/><nvpair id="nvpair" name="stonith-enabled" value="false"/><nvpair id="nvpair" name="stonith-action" value="reboot"/><nvpair id="nvpair" name="cluster-delay" value="180s"/><nvpair id="nvpair" name="default-action-timeout" value="120s"/><nvpair id="nvpair" name="no-quorum-policy" value="ignore"/><nvpair id="nvpair" name="symmetric-cluster" value="true"/><nvpair id="nvpair" name="short-resource-names" value="true"/><nvpair id="nvpair" name="is-managed-default" value="true"/><nvpair id="nvpair" name="stop-orphan-resources" value="true"/><nvpair id="nvpair" name="stop-orphan-actions" value="true"/><nvpair id="nvpair" name="remove-after-stop" value="false"/><nvpair id="nvpair" name="dc-deadtime" value="10s"/><nvpair id="nvpair" name="cluster-recheck-interval" value="0"/><nvpair id="nvpair" name="election-timeout" value="2min"/><nvpair id="nvpair" name="shutdown-escalation" value="20min"/><nvpair id="nvpair" name="crmd-integration-timeout" value="3min"/><nvpair id="nvpair" name="crmd-finalization-timeout" value="10min"/><nvpair id="nvpair" name="pe-error-series-max" value="-1"/><nvpair id="nvpair" name="pe-warn-series-max" value="-1"/><nvpair id="nvpair" name="pe-input-series-max" value="-1"/><nvpair id="nvpair" name="startup-fencing" value="true"/></cluster_property_set>
     </crm_config>
     <nodes>
       <node id="888e539a-c6ef-496d-b79a-77cbf0f9e5e4" uname="dl380g5b" type="normal"/>
       <node id="3831f23f-da0a-470e-b8e7-16e61a18c3c3" uname="dl380g5a" type="normal"/>
     </nodes>
     <resources>
       <master id="ms-sf"><meta_attributes id="master_slave-ms-sf.meta"><nvpair id="notify.meta.auto-77" name="notify" value="true"/><nvpair id="nvpair" name="clone-max" value="2"/><nvpair id="nvpair" name="clone-node-max" value="1"/><nvpair id="nvpair" name="master-max" value="1"/><nvpair id="nvpair" name="master-node-max" value="1"/></meta_attributes><group id="group"><meta_attributes id="group-group.meta"/>
           <primitive id="stateful-1" class="ocf" type="Stateful" provider="heartbeat"><meta_attributes id="primitive-stateful-1.meta"/>
             <operations>
               <op name="start" interval="0" id="start-1" timeout="60s" on-fail="stop"/>
               <op name="monitor" interval="10" id="monitor-1m" timeout="60s" on-fail="stop" role="Master"/>
               <op name="monitor" interval="20" id="monitor-1s" timeout="60s" on-fail="stop" role="Slave"/>
               <op name="stop" interval="0" id="stop-1" timeout="60s" on-fail="stop"/>
             </operations>
           </primitive>
           <primitive id="stateful-2" class="ocf" type="Stateful" provider="heartbeat"><meta_attributes id="primitive-stateful-2.meta"/>
             <operations>
               <op name="start" interval="0" id="start-2" timeout="60s" on-fail="stop"/>
               <op name="monitor" interval="10" id="monitor-2m" timeout="60s" on-fail="stop" role="Master"/>
               <op name="monitor" interval="20" id="monitor-2s" timeout="60s" on-fail="stop" role="Slave"/>
               <op name="stop" interval="0" id="stop-2" timeout="60s" on-fail="stop"/>
             </operations>
           </primitive>
         </group></master>
     </resources>
     <constraints>
       <rsc_location id="rsc_location" rsc="ms-sf">
         <rule id="location-0" role="Master" score="100">
           <expression id="expression" attribute="#uname" operation="eq" value="dl380g5a"/>
         </rule>
         <rule id="location-1" role="Master" score="500">
           <expression id="expression" attribute="#uname" operation="eq" value="dl380g5b"/>
         </rule>
       </rsc_location>
     </constraints>
   </configuration>
   <status>
     <node_state id="888e539a-c6ef-496d-b79a-77cbf0f9e5e4" uname="dl380g5b" crmd="online" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="lrm.auto-1">
         <lrm_resources id="lrm_resources">
           <lrm_resource id="stateful-1:0" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-1-0_monitor_0" operation="monitor" transition-key="3:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:7;3:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="2" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1-0_start_0" operation="start" transition-key="13:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;13:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="6" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1-0_promote_0" operation="promote" transition-key="14:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;14:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="9" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1-0_demote_0" operation="demote" transition-key="10:5:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:8;10:5:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="12" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
           <lrm_resource id="stateful-1:1" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-1-1_monitor_0" operation="monitor" transition-key="5:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:7;5:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="4" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
           <lrm_resource id="stateful-2:0" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-2-0_monitor_0" operation="monitor" transition-key="4:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:7;4:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="3" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2-0_start_0" operation="start" transition-key="16:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;16:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="7" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2-0_promote_0" operation="promote" transition-key="17:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;17:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="8" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2-0_stop_0" operation="stop" transition-key="3:5:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;3:5:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="13" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
           <lrm_resource id="stateful-2:1" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-2-1_monitor_0" operation="monitor" transition-key="6:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:7;6:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="5" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="transient_attributes.auto-1">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/><nvpair id="nvpair" name="fail-count-stateful-2:0" value="1"/><nvpair id="nvpair" name="fail-count-stateful-1:0" value="1"/></instance_attributes>
       </transient_attributes>
     </node_state>
     <node_state id="3831f23f-da0a-470e-b8e7-16e61a18c3c3" uname="dl380g5a" crmd="online" shutdown="0" in_ccm="true" ha="active" join="member" expected="member">
       <lrm id="lrm.auto-2">
         <lrm_resources id="lrm_resources">
           <lrm_resource id="stateful-1:1" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-1-1_monitor_0" operation="monitor" transition-key="10:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:7;10:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="4" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1-1_start_0" operation="start" transition-key="23:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;23:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="6" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-1-1_monitor_20000" operation="monitor" transition-key="24:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;24:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="8" crm_feature_set="2.0" rc-code="0" op-status="0" interval="20000" op-digest="873ed4f07792aa8ff18f3254244675ea"/>
           </lrm_resource>
           <lrm_resource id="stateful-2:0" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-2-0_monitor_0" operation="monitor" transition-key="9:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:7;9:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="3" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
           <lrm_resource id="stateful-2:1" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-2-1_monitor_0" operation="monitor" transition-key="11:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:7;11:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="5" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2-1_start_0" operation="start" transition-key="25:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;25:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="7" crm_feature_set="2.0" rc-code="0" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
             <lrm_rsc_op id="stateful-2-1_monitor_20000" operation="monitor" transition-key="26:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:0;26:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="9" crm_feature_set="2.0" rc-code="0" op-status="0" interval="20000" op-digest="873ed4f07792aa8ff18f3254244675ea"/>
           </lrm_resource>
           <lrm_resource id="stateful-1:0" type="Stateful" class="ocf" provider="heartbeat">
             <lrm_rsc_op id="stateful-1-0_monitor_0" operation="monitor" transition-key="8:3:0938a084-9056-4b8c-90f5-8a3005520b8e" transition-magic="0:7;8:3:0938a084-9056-4b8c-90f5-8a3005520b8e" call-id="2" crm_feature_set="2.0" rc-code="7" op-status="0" interval="0" op-digest="f2317cad3d54cec5d7d7aa7d0bf35cf8"/>
           </lrm_resource>
         </lrm_resources>
       </lrm>
       <transient_attributes id="transient_attributes.auto-2">
         <instance_attributes id="instance_attributes"><nvpair id="nvpair" name="probe_complete" value="true"/></instance_attributes>
       </transient_attributes>
     </node_state>
   </status>
 </cib>
